setup.py

```diff
@@ -888,15 +889,20 @@ def configure_extension_build():
     # Cross-compile for M1
     if IS_DARWIN:
         macos_target_arch = os.getenv('CMAKE_OSX_ARCHITECTURES', '')
+
         if macos_target_arch in ['arm64', 'x86_64']:
+            report(f"-- Building for Darwin target architecture {macos_target_arch}")
             macos_sysroot_path = os.getenv('CMAKE_OSX_SYSROOT')
             if macos_sysroot_path is None:
                 macos_sysroot_path = subprocess.check_output([
                     'xcrun', '--show-sdk-path', '--sdk', 'macosx'
                 ]).decode('utf-8').strip()
             extra_compile_args += ['-arch', macos_target_arch, '-isysroot', macos_sysroot_path]
             extra_link_args += ['-arch', macos_target_arch]
-
+        elif macos_target_arch == '':
+            report('-- No target architecture found.')
+        else:
+            report(f"-- Found Darwin target architecture {macos_target_arch}")
```

Comment:
This should be an error or a warning, right? I.e. `macos_target_arch` which is not either arm64 or x86_64 is ignored, isn't it?